src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/x64/reduce.cpp

```diff
@@ -1,217 +1,52 @@
-            const auto& funcInput = funcInputs[i];
-            ov::Tensor tensor;
-            if (reductionType == ngraph::helpers::ReductionType::Prod) {
-                tensor = ov::test::utils::create_and_fill_tensor(funcInput.get_element_type(), targetInputStaticShapes[i], 10, 5);
-                if (netPrecision == ElementType::f32) {
-                    auto *rawBlobDataPtr = static_cast<float *>(tensor.data());
-                    for (size_t i = 0; i < tensor.get_size(); ++i) {
-                        rawBlobDataPtr[i] /= 10.f;
-                    }
-                } else if (netPrecision == ElementType::bf16) {
-                    auto *rawBlobDataPtr = static_cast<ngraph::bfloat16 *>(tensor.data());
-                    for (size_t i = 0; i < tensor.get_size(); ++i) {
-                        rawBlobDataPtr[i] /= 10.f;
-                    }
-                }
-            } else {
-                tensor = ov::test::utils::create_and_fill_tensor(funcInput.get_element_type(), targetInputStaticShapes[i]);
-            }
-
-            inputs.insert({funcInput.get_node_shared_ptr(), tensor});
-        }
-    }
-
-private:
-    ngraph::helpers::ReductionType reductionType;
-    ElementType netPrecision;
+std::vector<std::vector<ov::test::InputShape>> inputShapes_dyn = {
+    {{{{1, 5}, 19, {1, 5}, {1, 10}}, {{2, 19, 2, 2}, {2, 19, 2, 9}}}},
 };
 
-TEST_P(ReduceCPULayerTest, CompareWithRefs) {
-    run();
+std::vector<std::vector<ov::test::InputShape>> inputShapes_5D_dyn = {
+    {{{{1, 5}, 19, {1, 5}, {1, 5}, {1, 5}}, {{2, 19, 2, 2, 2}, {2, 19, 3, 2, 2}}}},
+};
 
-    CheckPluginRelatedResults(compiledModel, "Reduce");
-}
-namespace {
-const std::vector<ElementType> inpOutPrc = {ElementType::bf16, ElementType::f32};
+std::vector<std::vector<ov::test::InputShape>> inputShapes_6D_dyn = {
+    {{{{1, 5}, 19, {1, 5}, {1, 5}, {1, 5}, {1, 5}}, {{2, 19, 2, 2, 2, 2}, {2, 19, 2, 2, 3, 2}}}},
+};
 
-const std::vector<bool> keepDims = {
-        true,
-        false,
+std::vector<std::vector<ov::test::InputShape>> inputShapes_Int32_dyn = {
+    {{{{1, 5}, 19, {1, 5}, {1, 10}}, {{2, 19, 2, 2}, {2, 19, 2, 3}}}},
 };
 
-const std::vector<std::vector<int>> axes = {
-        {0},
-        {1},
-        {2},
-        {3}
+std::vector<std::vector<ov::test::InputShape>> inputShapes_SmallChannel_dyn = {
+    {{{{1, 5}, 3, {1, 5}, {1, 10}}, {{2, 3, 2, 2}, {2, 3, 2, 9}}}},
 };
 
-const std::vector<std::vector<int>> axesND = {
-        {0, 1},
-        {0, 2},
-        {0, 3},
-        {1, 2},
-        {1, 3},
-        {2, 3},
-        {0, 1, 2},
-        {0, 1, 3},
-        {0, 2, 3},
-        {1, 2, 3},
-        {0, 1, 2, 3}
+std::vector<std::vector<ov::test::InputShape>> inputShapes_SingleBatch_dyn = {
+    {{{{1, 5}, 19, {1, 5}, {1, 10}}, {{1, 19, 2, 2}, {1, 19, 2, 9}}}},
 };
 
+std::vector<CPUSpecificParams> cpuParams_4D = {
+        CPUSpecificParams({nChw16c}, {nChw16c}, {}, {}),
+        CPUSpecificParams({nhwc}, {nhwc}, {}, {}),
```

Comment:
Here we explicitly test nhwc only for x86.
Don't we plan to fix / address the issue with nhwc for ACL?

Reply:
NHWC was disabled for ACL Executor explicitly due to accuracy issues:
https://github.com/openvinotoolkit/openvino/blob/0541a12730b31fe8811811a26dc016bfa4e79316/src/plugins/intel_cpu/src/nodes/reduce.cpp#L1875
Issue is raised in ACL repo: https://github.com/ARM-software/ComputeLibrary/issues/1044

Reply:
Agreed to move ifdef into SetUp() (using some util method) to skip the tests based on NHWC layout